rust/src/fees.rs

```diff
@@ -17,11 +17,13 @@ fn cbor_uint_length(x: u64) -> usize {
 }
 
 #[wasm_bindgen]
-pub fn min_fee(tx: &Transaction) -> usize {
-    const A: usize = 500;
-    const B: usize = 2;
-    let size = txsize(tx);
-    A * size + B
+pub fn min_fee(tx: &Transaction, coefficient: &Coin, constant: &Coin) -> Result<Coin, JsValue> {
+    let size = fees::txsize(tx) as u64;
+    size
+        .checked_mul(coefficient.unwrap())
```

Comment:
Since we have those methods on `Coin` now we could probably just do `Coin::new(size).checked_mul(coefficient)?.checked_add(constant)` instead of having to unwrap them and map back and forth.